bedrock/firefox/templates/firefox/releases/notes.html

```diff
@@ -210,6 +210,14 @@ <h3>“Firefox isn’t just fast and smooth, it also has lots of features and op
       </div>
     {% endif %}
 
+    {% if variation == 'c' %}
+      <div id="social-proof" class="mzp-l-content mzp-has-sidebar mzp-l-sidebar-left">
+        <div class="mzp-l-main mzp-l-article">
+          {{ download_firefox(alt_copy='Download now', download_location='secondary cta') }}
```

Comment:
_x-post from slack:_
The new social proof variant with no copy looks a little strange to me- like it was not intentional. Would it effect the test if we put something to the left? Like a word mark?

![image](https://user-images.githubusercontent.com/30483988/80135767-88ad6200-8555-11ea-9479-30045483f557.png)

I know the idea is to test whether it’s just the download button itself performing well vs. social proof copy, but usually it has some sort of context. I’m not going to block it, just wanted to mention.